lhotse/cut/set.py

```diff
@@ -2550,6 +2550,19 @@ def prefetch(self, buffer_size: int = 10) -> "CutSet":
             )
         )
 
+    def to_huggingface_dataset(self):
+        """
+        Converts a CutSet to a HuggingFace Dataset. Currently, only MonoCut with one recording source is supported.
+        Other cut types will be supported in the future.
+
+        More detailed description is in lhotse/hf.py
```

Comment:
One last comment: can you copy (duplicate) the documentation from `export_to_hf` here? It will be more user friendly for people using interactive suggestions (notebooks, IDEs)

Reply:
That's a good point. It's done.